lib/modules/dosomething/dosomething_kudos/dosomething_kudos.module

```diff
@@ -307,5 +307,23 @@ function dosomething_kudos_term_is_selected($reportback_item, $term) {
  * @return bool
  */
 function dosomething_kudos_disable_reactions($nid) {
-  return !variable_get('dosomething_reportback_kudos_enabled', FALSE) || dosomething_helpers_get_variable('node', $nid, 'disable_reactions');
+  return dosomething_helpers_get_variable('node', $nid, 'disable_reactions');
+}
+
+/**
+ * Grab all Kudos data required to render the markup for a given reportback fid
+ * @param  int $fid Reportback file id
+ * @return array Values needed to render Kudos markup
+ */
+function dosomething_kudos_get_kudos_data_for_fid($fid) {
+  $kudos = [];
```

Comment:
yar i asked before if that made sense but i think the comment chain got closed out before you had a chance to see it https://github.com/DoSomething/phoenix/pull/6689#discussion_r69950096